validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java

```diff
@@ -270,7 +276,15 @@ public void loadCredentials() {
       if (!hasAuthType()) {
         throw new RuntimeException("Credential cloud.auth_type definition missing");
       }
-      for (String keyFile : ALL_KEY_FILES) {
+      String authType = getAuthType();
+      Set<String> keyFiles;
```

Comment:
declare final or use ternary if

Reply:
Thanks for the suggestion, I corrected this to ternary if.